packages/stream_chat_flutter/lib/src/user_list_view.dart

```diff
@@ -76,6 +76,7 @@ class UserListView extends StatefulWidget {
   /// The query filters to use.
   /// You can query on any of the custom fields you've defined on the [Channel].
   /// You can also filter other built-in channel fields.
+  // TODO: Make it non-nullable in a future breaking release
```

Comment:
we already have breaking changes in the next release, don't we?

Reply:
not anymore, I deprecated it to deprecation instead
https://github.com/GetStream/stream-chat-flutter/pull/688